guvnor-ala/guvnor-ala-build-maven/src/main/java/org/guvnor/ala/build/maven/executor/MavenDependencyConfigExecutor.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.guvnor.ala.build.maven.executor;
+
+import java.util.Optional;
+import javax.inject.Inject;
+
+import org.eclipse.aether.artifact.Artifact;
+import org.guvnor.ala.build.maven.config.MavenDependencyConfig;
+import org.guvnor.ala.build.maven.model.MavenBinary;
+import org.guvnor.ala.build.maven.model.impl.MavenBinaryImpl;
+import org.guvnor.ala.config.Config;
+import org.guvnor.ala.pipeline.FunctionConfigExecutor;
+import org.guvnor.ala.registry.BuildRegistry;
+import org.kie.scanner.MavenRepository;
```

Comment:
This is deliberately a troll question... do we want to have guvnor-ala depend on drools?